apps/src/signUpFlow/LoginTypeSelection.tsx

```diff
@@ -39,6 +39,7 @@ const LoginTypeSelection: React.FunctionComponent = () => {
   const [showConfirmPasswordError, setShowConfirmPasswordError] =
     useState(false);
   const [showEmailError, setShowEmailError] = useState(false);
+  const [emailErrorMessage, setEmailErrorMessage] = useState('');
   const [email, setEmail] = useState('');
   const [authToken, setAuthToken] = useState('');
```

Comment:
Can this `authToken` be used instead of getting a new one on line 122?

Reply:
Oh I was just moving that one from lower in the function but thanks for pointing this out! Removed the line entirely.